package/MDAnalysis/analysis/pca.py

```diff
@@ -414,6 +414,118 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         return dot
 
+    def project_single_frame(self, components=None, group=None, anchor=None):
+        """Projects the structure onto selected PCs
+
+        Applies Inverse-PCA transform to the PCA atomgroup.
+        Optionally, calculates one displacement vector per residue
+        to extrapolate the transform to atoms not in the PCA atomgroup.
+
+        Parameters
+        ----------
+        components : int, array, optional
+            Components to be projected onto.
+            The default ``None`` maps onto all components.
+
+        group : AtomGroup, optional
+            The AtomGroup or Universe containing atoms to be projected.
+            The atoms in the PCA class will be projected regardless.
+
+        anchor : string, optional
+            String to select the atom whose displacement vector is applied to
+            non-PCA atoms in a residue. The atomselection must have exactly
+            one PCA atom in each residue of group. The atomselection must
+            not have any atoms thats were not part of the PCA transformation.
+            The default ``None`` does not extrapolate the projection
+            to non-PCA atoms. anchor cannot be ``None``
+            if group is not ``None``.
+
+        Returns
+        -------
+        function
+            The projecting function which transforms a timestep.
```

Comment:
Document the signature of the returned function here.

Reply:
I wasn't clear here, I think: The return value *is* a function, so your original docs were correct., What I meant was to add in the text ("The projecting function ...") to write something like: The resulting function `f` takes as input a `Timestep` ts, `f(ts)` and returns the Timestep. 

Or something like that. It should be clear that it can be used as a transformation.